nowcasting_dataset/config/example.yaml

```diff
@@ -12,8 +12,8 @@ output_data:
   filepath: prepared_ML_training_data/v4/
 process:
   batch_size: 32
-  forecast_length: 12
-  history_length: 6
+  forecast_minutes: 60
```

Comment:
Yea, I know what you mean. It would be nice to define a time delta, as in the future it could be 8.5 hours, and we don't want to be writing that in minutes. But writing things we just simple minutes does make it simple. 

Seems a trade off of how simple we want to make it, and right now, seems ok to keep it like this